Dockerfile

```diff
@@ -1,13 +1,18 @@
-FROM python:3.10-slim-buster
+FROM python:3.10-slim-bookworm as base
 RUN apt-get update > /dev/null && apt-get install -y --no-install-recommends \
-    build-essential=12.6 \
-    curl=7.64.0-4+deb10u5 \
-    file=1:5.35-4+deb10u2 \
-    git=1:2.20.1-2+deb10u3 \
-    default-libmysqlclient-dev=1.0.5 \
-    mecab=0.996-6 \
-    mecab-ipadic-utf8=2.7.0-20070801+main-2.1 \
-    libmecab-dev=0.996-6 \
-    swig=3.0.12-2 \
+    build-essential=12.9 \
+    curl=7.88.1-10+deb12u4 \
+    file=1:5.44-3 \
+    git=1:2.39.2-1.1 \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
+FROM base as dictionary-builder
```

Comment:
I didn't understand the reason for making it multi-stage.
In particular, I think it's wasteful to do `apt-get update` after doing `apt-get clean`.

Reply:
@phirose 
I've confirmed `mecab=0.996-6`,`libmecab-dev=0.996-6` to build  `mecab-ipadic-neologd`.
But I'd like to exclude mecab=0.996-6,libmecab-dev=0.996-6 in Python-mecab-builder image to install  version 1.0.8  with pip in `user-app-server-side`.
Considering these things I  thought the multi stage is required.
Do you have the other idea?

Reply:
This is a builder and will not be included in the final user-app-server-side image.
Therefore, I don't think there is a need to be overly concerned about minimizing.
Also, those packages have nothing to do with mecab-python3 installation. It does not need to be included in the final image as it is confusing, but it has no effect if included in the middle.

As I wrote in another comment, it would be simpler to not use this image for `pip install`.
https://github.com/bebit/user-app-server-side/pull/1950/files#r1435916210